src/utils/camelCase.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import camelcaseKeys from 'camelcase-keys';
+
+export const camelCase = (data: any, options: any) => {
+  return camelcaseKeys(data, options);
+};
```

Comment:
Is there a specific reason why we’re wrapping it like this here? It doesn’t appear to offer any particular advantage when exporting this.